release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1474,8 +1474,16 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |Removed
 |Removed
-|====
 
+|Shared Resources CSI Driver ^[1]^
+|Deprecated
+|Deprecated
+|Deprecated 
+|====
+[.small]
+--
+1. The Shared Resource CSI Driver feature is now generally available in Builds for Red Hat OpenShift 1.1. This feature is now deprecated in OpenShift Container Platform. To use this feature, ensure you are using Builds for Red Hat OpenShift 1.1 or a more recent version.
+--
```

Comment:
Is there a way to call out this information without this footnote + style?

Reply:
Hi @maxwelldb , please see table 9 in https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-deprecated-removed-features

We decided to follow the same method.